### DISCUSSION TITLE
Linked zoom rate w/ 2 views

### DISCUSSION THREAD

**Discussion by entervine** [pelicanmapping/osgearth#2525] [Support Q&A]
OSGEarth 2.10, OSG 3.4.1
In my application, I have two different viewpoints on one view. I have a call 'setViewPoint' which is activated via a button press and it sets the viewpoint of the manipulator. For simplicity i'll call this viewpoint1 and viewpoint2. On viewpoint1, zoom way in (next to the ground) then go to viewpoint2 and zoom way out. Now go back to viewpoint1 and zoom in one click. Next go to viewpoint2 and try to zoom in. You'll see a slow zoom but the camera stays zoomed out, at some point it quits moving. Then if you go to viewpoint1, then back to viewpoint2 you'll notice you're next to the earth.

TL;DR, if you switch between viewpoints with significant differences in ranges the camera gets out of whack. I suspect there is an easy one line fix for this, but I am stuck.

It's also worth noting when zooming back in on viewpoint1, it's zooming towards the center of viewpoint1.

**Comment by gwaldron** [comment]
Do you experience the same issue in osgEarth 3.5?
